src/value.cc

```diff
@@ -657,14 +657,6 @@ RangeType Value::toRange() const
   else return RangeType(0,0,0);
 }
 
-Value &Value::operator=(const Value &v)
```

Comment:
@kintel do you have any concerns removing this one? Newer compilers complain a lot with just the assignment operator but no custom copy constructor.

Reply:
The rule of zero says to remove both that and the empty destructor in value.h.  It looks like the defaults should work fine.

Reply:
Oh, good point, I missed the destructor. I've pushed that to the `function-literals` branch to get it *soon-ish*...